erpnext/accounts/general_ledger.py

```diff
@@ -288,6 +288,7 @@ def get_merge_properties(dimensions=None):
 		"project",
 		"finance_book",
 		"voucher_no",
+		"against",
```

Comment:
@ruthra-kumar,

Is this acceptable?

Reply:
With this, it will be one unique row for each against account, and never a composite value.